src/Connector.js

```diff
@@ -133,8 +135,10 @@ class Connector {
     await promisify(this.client, 'unregistered', this.client.unregister.bind(this.client), id);
   }
 
+  // TODO? what did the old { type: 'knot:thing' } query parameter do?
```

Comment:
`thing`, `gateway` and `app` are concepts from the Meshblu version. We don't need to concern about this query anymore, only `things` are returned on this operation.